src/Simplex/Chat.hs

```diff
@@ -462,14 +465,18 @@ processChatCommand = \case
   APIActivateChat -> withUser $ \_ -> do
     restoreCalls
     withAgent foregroundAgent
-    withStoreCtx' (Just "APIActivateChat, getUsers") getUsers >>= void . forkIO . startFilesToReceive
+    users <- withStoreCtx' (Just "APIActivateChat, getUsers") getUsers
+    chatReadVar subscriptionMode >>= \case
+      SMOnlyCreate -> void . forkIO $ subscribeUsers True users
+      SMSubscribe -> pure () -- Subscriptions are automatic, nothing to do
```

Comment:
```suggestion
    chatReadVar subscriptionMode >>= \case
      SMOnlyCreate -> pure ()
      SMSubscribe -> void . forkIO $ subscribeUsers True users
```

It's actually other way around, you want to subscribe new subscriptions only in SMSubscribe mode.